### ISSUE TITLE
unsupported dependencies in LTS 7

### ISSUE THREAD

**Issue by DanielRuf** [YAG-Gallery/yag#59]
```
Benötigte Erweiterung "pt_extlist":
Your TYPO3 version is higher than this extension requires. It requires TYPO3 versions 6.2.0 - 7.0.99
Benötigte Erweiterung "pt_extbase":
Your TYPO3 version is higher than this extension requires. It requires TYPO3 versions 6.2.0 - 7.0.99
```

Are they required and how can we get this to work with LTS 7.6.x?

**Comment by DanielRuf** [comment]
Still open.

**Comment by daniellienert** [comment]
All dependencies are available compatible for TYPO3 version 7.6+
